zkevm-circuits/src/witness/mpt.rs

```diff
@@ -235,16 +250,17 @@ impl MptUpdate {
 
 #[derive(Eq, PartialEq, Hash, Clone, Debug, Copy, PartialOrd, Ord)]
 enum Key {
+    Account {
+        address: Address,
+        field_tag: AccountFieldTag,
+    },
     AccountStorage {
+        // god what are these fields,t even?
         tx_id: usize,
         address: Address,
         storage_key: Word,
         exists: bool,
     },
-    Account {
-        address: Address,
-        field_tag: AccountFieldTag,
-    },
```

Comment:
why change the AccountStorage / Account order here? @z2trillion

Reply:
i switched these so that the accountstorage updates would be at the end of the sorted list of mpt updates. 

This change isn't needed needed to integrate the mpt circuit into the super circuit.